src/app/useDeepLinks.ts

```diff
@@ -64,7 +68,7 @@ export const useDeepLinks = () => {
     CleverTap.getInitialUrl(async (err: any, url: string) => {
       if (err) {
         if (/CleverTap initialUrl is (nil|null)/gi.test(err)) {
-          Logger.warn('useDeepLinks/useAsync', 'CleverTap InitialUrl is nil|null', err)
+          Logger.debug('useDeepLinks/useAsync', 'CleverTap InitialUrl is nil|null', err)
```

Comment:
this is expected when the app is open normally and not from a clevertap notification, so this doesn't need to be a warn